addons/website_event/controllers/main.py

```diff
@@ -154,6 +154,9 @@ def event_page(self, event, page, **post):
         if '.' not in page:
             page = 'website_event.%s' % page
 
+        # Every event page view should have it's own SEO
+        values['seo_object'] = request.website.viewref(page)
```

Comment:
should move that after the `try except` I think because in case of fallback you will not have the right one

Reply:
Good catch, actually inside the try will even be better so we don't set 404 in seo_object for nothing (will work without it)